CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## v0.0.1 - 2022-01-11 - Moving
+## v0.0.2 - 2022-10-27
+
+* Enable support for root level NS records (`SUPPORTS_ROOT_NS=true`)
+
+## v0.0.1 - 2022-01-11
```

Comment:
Looks like the "title" of this release was removed as part of adding the stuff up above.
```suggestion
## v0.0.1 - 2022-01-11 - Moving
```